app/validators/bill-runs/setup/type.validator.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+/**
+ * Validates data submitted for the `/bill-runs/setup/{sessionId}/type` page
+ * @module TypeValidator
+ */
+
+const Joi = require('joi')
+
+const VALID_VALUES = [
+  'annual',
+  'supplementary',
+  'two_part_tariff',
+  'two_part_tariff_supplementary'
```

Comment:
This is not in the list of options on the page. is it going to be added later?

Reply:
Good point. It is coming. But I couldn't be 100% certain that is what the name of the bill run type will be when we do have to implement it.

So, on reflection, it would be better that we add this as a valid bill run type when we make the changes to support it.